05_DeleteUser.php

```diff
@@ -55,19 +54,18 @@
 		
 
 
-
 			<div style="margin-left:8%">
 				<div class="w3-container w3-dark-grey">
-			 	 <h1>Delete User</h1>
+			 	 <h2>Delete User</h2>
```

Comment:
changed from h1 to h2; creating the same theme as the other admin titles for each query